### PR TITLE
open-cluster-management/backlog#14928 ki ans auto rerun

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -670,6 +670,30 @@ See the following limitations to various _Application_ tables in the console:
...
+
+. Run the following command:
++
+----
+oc edit kubeapiservers.operator.openshift.io cluster
+----
+
+. Find the `unsupportedConfigOverrides` section.
+
+. Update the value of `TTLAfterFinished` to `false` to disable the job cleanup feature. This update resembles the following example:
```
Comment (gparvin) [inline release_notes/known_issues.adoc]: The contents prior to editing most likely are going to look like:

```
  unsupportedConfigOverrides:  null
```

So the `TTLAfterFinished` is new content and not a field being added. 

Also see my follow-on comment.  The same edit needs to be performed for `oc edit kubecontrollermanager cluster` also.

**Review by gparvin — COMMENTED** [review]
Added comment to new known issue proposal.  Thanks!

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -670,6 +670,30 @@ See the following limitations to various _Application_ tables in the console:
 [#governance-known-issues]
 == Governance known issues
 
+[#ans-job-cleanup-restarts-jobs]
+=== Ansible Automation jobs continue to run hourly even though no new policy violations started the automation
+//2.3:14928
+
+In {ocp-short} 4.8 the TTL Controller for Finished Resources is enabled by default, which means jobs are removed hourly. This job cleanup causes the Ansible Automation Platform Resource Operator to rerun the associated automation. The automation runs again with the existing details in the AnsibleJob resource that was created by the policy framework. The details provided might include previously identified violations, which can mistakenly appear that the violation was repeated. You can disable the controller that cleans up the jobs to prevent these duplicate violations. To disable the controller that cleans up the jobs, complete the following steps:
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: `AnsibleJob` resource


..mistakenly appear as a repeated violation.

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -670,6 +670,30 @@ See the following limitations to various _Application_ tables in the console:
...
+
+. Run the following command:
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: Run the following command to edit the `kubeapiservers.operator.openshift.io` resource:

**Review by dockerymick — CHANGES_REQUESTED** [review]
Thanks for taking this Chris, i made a few comments.

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: Sorry Gus. I missed your comment. I have now added it.

**Comment by cadawson** [comment]
Thanks for the reviews. The PR is updated.

**Review by dockerymick — APPROVED** [review]
looks good thanks!

**Review by gparvin — APPROVED** [review]
Thanks

**Pull request merged**